### DISCUSSION TITLE
Next.js with TailwindCSS is unusably slow in development

### DISCUSSION THREAD

**Discussion by tunztunztunz** [vercel/next.js#25433] [Help]
I have an extremely basic Next.js site that uses TailwindCSS and it's so slow that I can hardly load pages. Sometimes it takes upwards of 33s to load or they don't load at all between routes. I'm not really sure what I can do to fix this.

**Comment by bobarros** [comment]
And what are your thoughts about how the community could help you?

**Reply by tunztunztunz** [reply]
I can't be the only person using TailwindCSS and Nextjs experiencing these issues. So, I wanted to see if someone could point me to a solution or make the issue known.

**Comment by SahasSaurav** [comment]
just turn tailwind jit for development 
https://tailwindcss.com/docs/just-in-time-mode#enabling-jit-mode
but tailwind jit is currently in alpha so turn off before depolyment

**Reply by tunztunztunz** [reply]
I tried this earlier today and it didn't make a difference, but I just enabled it again and it seems to help a bit. I still experience 1-3 second load times between hard refreshes and routing, but it's a huge improvement. What are the dangers of leaving it enabled in production?

**Reply by SahasSaurav** [reply]
As tailwind team  also suggest it jit in alpha version so syntax or code might change
